app/controllers/hub/dashboard_controller.rb

```diff
@@ -128,5 +140,55 @@ def capacity_count
         0
       end
     end
+
+    def load_return_summaries
```

Comment:
are there any (substantial) conditional paths through this method that aren't covered by the test cases you added to the controller spec?